README.md

```diff
@@ -24,33 +24,33 @@ cd mpact-compiler
 git submodule update --init --recursive --progress
 ```
 
-All following commands assume you remain in the `mpact-compiler` directory.
+NOTE: All of the following commands assume you remain in the `mpact-compiler` directory.
 
 ### Setup Python virtual environment
 
-The following commands initialize a virtual environment.
+The following commands initialize a virtual environment under bash/sh/etc.  For other shells, see Note 1, [below](README.md#notes).
```

Comment:
```suggestion
The following commands initialize a virtual environment under bash/sh/etc. For other shells, see Note 1, [below](README.md#notes).
```

Reply:
This is a generational thing -- old people who used _typewriters_(!) put in two spaces following a period b/c typewriters, by definition, are monospace, and that was the way to distinguish the end of a sentence.  (There's some evidence that (even before typewriters) typesetters put in more than one (but less than two?) spaces between sentences.)  (Also please note that both of these parentheticals are separated by two spaces from their predecessors, and I bet(?) you didn't notice... :-)

I haven't updated my typing because: a) I'm old and stubborn/lazy (part i of a: b/c this is clearly a generational issue, which means you're being ageist :-); part ii of a: as you will someday learn, old eyes appreciate extra help, and cramming stuff together is the opposite of "help" :-), b) I believe (in rare cases) it is a parsing aid, b/c it distinguishes the end of a sentence from an abbreviation (e.g., https://www.reddit.com/r/grammar/comments/161gp77/i_found_a_good_example_of_why_two_spaces_after_a/?rdt=36234) without ever(!) removing clarity (so there's no downside), c) (and I only just found this, so it isn't really an argument I had before typing this (silly) response :-) a study (https://www.forbes.com/sites/tarahaelle/2018/05/09/one-or-two-spaces-after-a-period-that-new-study-cant-tell-you/) says (with caveats :-) there's a 3% increase in reading speed for using the double space, d) the one-space-is-clear argument only holds for variable-width fonts, which, as you can see above, is not what we program in, and e) even if you tell me there's some "style requirement" that says I have to do it someone else's way, I didn't vote for them, so...no.  :-)

Reply:
Honestly, I have no idea that it's a generational thing, so thanks for letting me know! My only reservation is consistency. Since the existing file used single space after period (see the first paragraph), if you think double space helps and Aart is okay with changing it, I'm totally okay. ;)

Reply:
I am not a fan of the two space convention, honestly ;-)